node/src/neighborhood/mod.rs

```diff
@@ -1434,6 +1693,230 @@ impl Neighborhood {
         undesirability + node_undesirability
     }
 
+    fn handle_exit_location_message(
+        &mut self,
+        message: UiSetExitLocationRequest,
+        client_id: u64,
+        context_id: u64,
+    ) {
+        let (exit_locations_by_priority, missing_locations) =
```

Comment:
Jinde kdyz pouzivas to slovo  "locations" referujes o necem jinem nez pouze Stringu s tim country kodem. Tady ale ty data jsou fakticky pouze string. Mozna by to chtelo prejmenovat. At to neni zbytecne zavadejici. Mozna proste "missing_countries"